addons/web/static/src/less/fields.less

```diff
@@ -21,6 +21,20 @@
     white-space: nowrap;
 }
 
+// less properties for copy widget
+.o_field_copy {
+    position: relative;
+    width: 100% !important;
```

Comment:
using !important is usually a problem. please try to remove it

Reply:
@ged-odoo if we remove !important then it will be overridden by o_field_widget width. thanks